man/fi_mr.3.md

```diff
@@ -546,6 +565,27 @@ region.  The domain authorization key will be used if the auth_key_size
 provided is 0.  This field is ignored unless the fabric is opened with API 
 version 1.5 or greater.
 
+## iface
+Indicates the software interfaces used by the application to allocate and
+manage the memory region. This field is ignored unless the fabric is opened
+with API version 1.9 or greater.
+
+*FI_HMEM_SYSTEM*
+: Uses standard operating system calls and libraries, such as malloc,
+  calloc, realloc, mmap, and free.
+
+*FI_HMEM_CUDA*
+: Uses Nvidia CUDA interfaces such as cuMemAlloc, cuMemAllocHost,
+  cuMemAllocManaged, cuMemFree, cudaMalloc, cudaFree.
+
+## device
+Reserved 64 bits for device identifier if using non-standard HMEM interface.
+This field is ignored unless the fabric is opened with API version 1.9 or
```

Comment:
Please update to indicate that the field is ignored unless the iface field is valid.